### PR TITLE
Remove junit 4 entirely

### PR DESCRIPTION
**Pull request by sokomishalov**
https://github.com/lettuce-io/lettuce-core/issues/1478

### CODE REVIEW

File: src/test/java/io/lettuce/core/cluster/ClusterSetup.java [hunk 1]
```diff
@@ -38,7 +38,7 @@
      *
      * @param clusterRule
      */
-    public static void setup2Masters(ClusterRule clusterRule) {
+    public static void setup2Masters(ClusterTestHelper clusterRule) {
```
Comment (dengliming) [inline src/test/java/io/lettuce/core/cluster/ClusterSetup.java]: should we also rename `clusterRule ` to `clusterTestHelper`?(maybe)

Comment (sokomishalov) [reply src/test/java/io/lettuce/core/cluster/ClusterSetup.java]: Absolutely yes, fixed.

**Review by dengliming — APPROVED** [review]
LGTM

**Review by mp911de — APPROVED** [review]

**Comment by mp911de** [comment]
Thank you for your contribution. That's merged, polished, and backported now.

**Pull request closed without merging**